freeipa/src/main/java/com/sequenceiq/freeipa/controller/FreeIpaV1Controller.java

```diff
@@ -160,6 +160,12 @@ public DescribeFreeIpaResponse describeInternal(@ResourceCrn String environmentC
         return freeIpaListService.list(accountId);
```

Comment:
you can use `listInternal` method here to ensure if both list logic give the same result

Reply:
done.
I was worried that this may interact with the @InternalOnly annotation, but it doesn't. I'm guessing that the annotations are used to create a proxy when called by outside classes but won't affect calls from within the controller class.

Reply:
yes, spring won't proxy calls within class, thus aspect cannot catch the call and authorization won't happen once again.